sofiadinizms/vida-de-inseto
# Feature/level 1

## What I did
- Created the Level One general mechanics: touch balloon to see tip, shake device to show food, shake again so the character eats the food.
- Added temporary assets to demonstrate said mechanics

## How to test 
- Git pull this branch
- Run it on an Xcode simulator
- Check if a mushroom and a thought balloon appear on the center of the screen
- When you touch the balloon, an iOS alert should show up (in the future it will contain the tip)
- Use the keyboard shortcut command + control + Z to simulate a shake 
- Wait a second for the fruit to show up
- Use the shortcut again to make the fruit fall into the character 

If you managed to successfully complete all these tasks as described above, then everything is working correctly :)
